crates/core/component/ibc/src/prefix.rs

```diff
@@ -0,0 +1,34 @@
+use ibc_types::core::commitment::MerklePrefix;
+use once_cell::sync::Lazy;
+
+/// The substore prefix used for IBC data.
+pub static IBC_SUBSTORE_PREFIX: &'static str = "ibc-data/";
+
+/// The IBC commitment prefix used for the IBC substore, as a [`MerklePrefix`].
+pub static IBC_COMMITMENT_PREFIX: Lazy<MerklePrefix> = Lazy::new(|| MerklePrefix {
+    key_prefix: IBC_SUBSTORE_PREFIX.as_bytes().to_vec(),
+});
+
+/// the ICS23 proof spec for penumbra's IBC state; this can be used to verify proofs
+/// for other substores in the penumbra state, provided that the data is indeed inside a substore
+/// (as opposed to directly in the root store.)
+pub static IBC_PROOF_SPECS: Lazy<Vec<ics23::ProofSpec>> = Lazy::new(|| {
+    vec![
+        penumbra_storage::ics23_spec(),
```

Comment:
I'm wondering if we should move the proof spec to the storage crate, but this is a bit tricky since the correct proof spec to use now depends on the key (whether or not it's in a substore)

Reply:
I don't think it should move to the storage crate, I think we should say that this is the proof spec for the IBC data only.